src/pages/documentation/adobe-journey-optimizer/release-notes.md

```diff
@@ -6,10 +6,10 @@
 
 * Adds support for Adobe Journey Optimizer powered in-app messages.
 
-### Android AEPMessaging 2.0.1
+### Android Messaging 2.0.1
 
 * Adds support for Adobe Journey Optimizer powered in-app messages.
-* Adds support for AEP 2.x extensions. Note, AEPMessaging 2.0.1 is not backwards compatible with the ACP 1.x extensions.
+* Adds support for AEP 2.x extensions. Note, Messaging 2.0.1 is not backwards compatible with the ACP 1.x extensions.
```

Comment:
I'm a little confused here, shouldn't this be for 2.0.0?

Reply:
I am going to remove this line.